sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/BlobServiceVersion.java

```diff
@@ -33,4 +33,18 @@ public String getVersion() {
     public static BlobServiceVersion getLatest() {
         return V2019_02_02;
     }
+
+    /**
+     * Gets the enum BlobServiceVersion from version string
+     * @param serviceVersion the string value of the service version.
+     * @return
+     */
+    public static BlobServiceVersion getBlobServiceVersion(String serviceVersion) {
+        for(BlobServiceVersion version : values()){
```

Comment:
Don't need to loop through the values each time this method is called. The values can be cached in a static map and looked up in this method.